src/modules/github.cpp

```diff
@@ -64,9 +70,12 @@ namespace modules {
       content = request();
     } catch (application_error& e) {
       m_log.warn("%s: cannot complete the request to github: %s", name(), e.what());
```

Comment:
I didn't think about that before, but since "offline" is now a normal state, maybe we should replace this line by:
```suggestion
      m_log.info("%s: cannot complete the request to github: %s", name(), e.what());
```

What do you think of that @patrick96?

Reply:
Good idea, we don't want to spam users with warnings that aren't really warnings.